### PR TITLE
[Bugfix:Calendar] input boxes for calendar events

### LINKED ISSUE #9249: Calendar input fields don't line up when creating item.
**What problem are you trying to solve with Submitty**
The input fields don't line up. I think the message text field should also be larger to accommodate a longer message without the user having to scroll.

**Describe the way you'd like to solve this problem**
Adding a margin or padding on the left to make them line up should be simple enough. Making the message text a _text area_ may be useful. Maybe making the order:
1. Type
2. Date
3. <TEXTAREA> Message Text

**Describe any potential alternatives you'd tried to solve the problem**

**Additional context**

### PR DESCRIPTION
**Pull request by joonbang**
Fixes #9249. 

Rearranges text boxes and adds CSS to restructure input elements for calendar events. Allows for larger input submission by swapping <input> to <textarea>. 

Before:
232245564-965644eb-0492-4558-bf3b-351827dbcee6

After:
image

### CODE REVIEW

**Comment by williamjallen** [comment]
Hi @joonbang, thanks for this contribution.  Please add before and after screenshots to your PR description to make it easier for people to understand exactly what you are fixing here.

File: site/app/templates/calendar/NewCalendarItem.twig [hunk 1]
```diff
@@ -5,16 +5,31 @@
...
+
+    <style>
+        div.calendar-submit {
+            display: grid;
+            grid-template-columns: max-content max-content;
+            grid-gap: 5px; 
+        }
+        div.calendar-submit label { text-align: "right"; }
+        div.calendar-subm1t label:after { content: ":"; }
+    </style>
```
Comment (williamjallen) [inline site/app/templates/calendar/NewCalendarItem.twig:17]: CSS like this should be moved to the relevant CSS file.  `site/public/css/calendar.css` would probably be an appropriate place for this.

**Review by williamjallen — CHANGES_REQUESTED** [review]

File: site/app/templates/calendar/NewCalendarItem.twig (hunk 1 above)
Comment (joonbang) [reply site/app/templates/calendar/NewCalendarItem.twig]: Gotcha, have moved all relevant portions to the CSS file!

**Comment by joonbang** [comment]
Thanks for the feedback! I've modified the PR accordingly (with images & updated CSS).

**Comment by williamjallen** [comment]
@joonbang It looks like some of the tests failed.  Once the failing tests are resolved, I think this PR should be ready to merge.

**Pull request merged**